### PR TITLE
Fix PhpStorm findings

### PR DESCRIPTION
**Pull request by franmomu**
Not sure if this should be in the changelog. My plan after this was to do this in master branch:
- Fix Command deprecations (extending from `ContainerAwareCommand`) 
- Changing classes to final, add types, etc
- Add support for Symfony 5 (and dropping 3.4?)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     },
     "require-dev": {
         "doctrine/orm": "^2.2",
-        "symfony/phpunit-bridge": "^4.2"
+        "symfony/phpunit-bridge": "^4.3.8"
```
Comment (core23) [inline composer.json]: Can we use version 5?

File: src/Command/GenerateCommand.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $namespace = $input->getOption('namespace');
         if ($namespace) {
             if (!preg_match('/^(?:(?:[[:alnum:]]+|:vendor)\\\\?)+$/', $namespace)) {
-                throw new \InvalidArgumentException('The provided namespace \'%s\' is not a valid namespace!', $namespace);
+                throw new \InvalidArgumentException(sprintf('The provided namespace \'%s\' is not a valid namespace!', $namespace));
```
Comment (core23) [inline src/Command/GenerateCommand.php]:
```suggestion
                throw new \InvalidArgumentException(sprintf("The provided namespace '%s' is not a valid namespace!", $namespace));
```

File: src/Command/GenerateCommand.php [hunk 3]
```diff
@@ -68,7 +68,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $namespace = $input->getOption('namespace');
         if ($namespace) {
             if (!preg_match('/^(?:(?:[[:alnum:]]+|:vendor)\\\\?)+$/', $namespace)) {
-                throw new \InvalidArgumentException('The provided namespace \'%s\' is not a valid namespace!', $namespace);
+                throw new \InvalidArgumentException(sprintf("The provided namespace '%s' is not a valid namespace!", $namespace));
```
Comment (greg0ire) [inline src/Command/GenerateCommand.php]: Let's add some linebreaks and use double quotes:

```suggestion
                throw new \InvalidArgumentException(sprintf(
                    'The provided namespace "%s" is not a valid namespace!', 
                    $namespace
                ));
```

**Comment by core23** [comment]
If you can fix all phpstan findings, can you add it as a GH action?

**Comment by greg0ire** [comment]
@core23 it says phpstORM in the title

**Comment by core23** [comment]
Too early in the morning :laughing:

**Comment by franmomu** [comment]
> If you can fix all phpstan findings, can you add it as a GH action?

I'm going try it

**Comment by greg0ire** [comment]
> I'm going try it

wait so does this has to do with phpstorm, phpstan or both?

**Comment by franmomu** [comment]
> > I'm going try it
> 
> wait so does this has to do with phpstorm, phpstan or both?

I was for phpstorm at first, but fixing that also fixed some phpstan issues I believe. Should I open another PR with just the last commit?

**Comment by OskarStark** [comment]
No need for this overhead.
Ready to merge?

File: tests/Command/Functional/Entity/Block.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+namespace Sonata\EasyExtendsBundle\Tests\Command\Functional\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class Block
```
Comment (phansys) [inline tests/Command/Functional/Entity/Block.php]:
```suggestion
final class Block
```

File: tests/Command/Functional/DumpMappingCommandTest.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+{
+    public function testExecute(): void
+    {
+        $kernel = static::createKernel();
+        $application = new Application($kernel);
+        $command = $application->find('sonata:easy-extends:dump-mapping');
+        $commandTester = new CommandTester($command);
+        $commandTester->execute([
+            'manager' => 'default',
+            'model' => 'Sonata\EasyExtendsBundle\Tests\Command\Functional\Entity\Block',
```
Comment (phansys) [inline tests/Command/Functional/DumpMappingCommandTest.php]: Could we import this class and use `Block::class` instead?

**Comment by franmomu** [comment]
The entity classes [can't be `final`](https://github.com/doctrine/common/pull/771) because doctrine can't create proxy classes extending from them

**Comment by greg0ire** [comment]
Thanks @franmomu !

**Pull request merged**